### ISSUE TITLE
Adwaita icon theme crash on startup

### ISSUE THREAD

**Issue by natemaia** [nwg-piotr/sway-gtk-menu#41]
When starting the application with the adwaita icon theme as the system default, it crashes with the following traceback

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/sgtk_menu/grid.py", line 498, in app_image
    pixbuf = icon_theme.load_icon(icon, args.s, Gtk.IconLookupFlags.FORCE_SIZE)
gi.repository.GLib.Error: gtk-icon-theme-error-quark: Icon 'chrome-nmmhkkegccagdldgiimedpiccmgmieda-Default' not present in theme Adwaita (0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/sgtk-grid", line 11, in <module>
    load_entry_point('sgtk-menu==1.2.1', 'gui_scripts', 'sgtk-grid')()
  File "/usr/lib/python3.8/site-packages/sgtk_menu/grid.py", line 164, in main
    list_entries()
  File "/usr/lib/python3.8/site-packages/sgtk_menu/grid.py", line 422, in list_entries
    all_apps.append(AppBox(item[0], item[1], item[2], item[3]))
  File "/usr/lib/python3.8/site-packages/sgtk_menu/grid.py", line 465, in __init__
    self.button.set_image(app_image(icon))
  File "/usr/lib/python3.8/site-packages/sgtk_menu/grid.py", line 501, in app_image
    pixbuf = icon_theme.load_icon('help', args.s, Gtk.IconLookupFlags.FORCE_SIZE)
gi.repository.GLib.Error: gtk-icon-theme-error-quark: Icon 'help' not present in theme Adwaita (0)
```
**To Reproduce**
1. Set system icon theme to Adwaita
2. Run `sgkt-grid`

**Environment (please complete the following information):**
 - Linux distribution: Arch
 - Window Manager: yaxwm
 - Window Manager version: 0.1
 - Python version: 3.8

**Comment by nwg-piotr** [comment]
Well, use of `help` as a fallback icon was not the best idea. I think I need to provide an own .svg file. BTW: my Adwaita contains the "help" icon. :)

**Comment by nwg-piotr** [comment]
Resolved in [v1.2.4](https://github.com/nwg-piotr/sgtk-menu/releases/tag/v1.2.4). Thanks for reporting!
